blueocean-personalization/package.json

```diff
@@ -35,7 +35,7 @@
     "react-addons-test-utils": "15.3.2"
   },
   "dependencies": {
-    "@jenkins-cd/blueocean-core-js": "0.0.120",
+    "@jenkins-cd/blueocean-core-js": "0.0.121-orga",
```

Comment:
@alvarolobato changes to shrinkwrap are missing for web, personalization and dashboard modules. They should have pointer to 0.0.121-orga. See https://github.com/jenkinsci/blueocean-plugin/blob/master/blueocean-core-js/README.md#developing-with-prerelease-versions on how to develop with pre release version of core js.